client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -87,7 +87,7 @@ const RobotContainer = (props) => {
       {!hoveredUpon && (
         <Col className={[styles.box, styles.robotBox]}>
           <Row justify='center' align='middle' style={{ height: '100%' }}>
```

Comment:
Please add this to align the Heading in the Box.
```suggestion
          <Row justify='center' align='middle' style={{ height: '100%', margin:'0.5rem' }}>
```

Reply:
+1